client/Pages/Home/Home.spec.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+/* eslint-disable react/jsx-key */
+jest.mock('react-i18next');
+import { render } from '@testing-library/react';
+import React from 'react';
+import { Home } from '.';
+import { translate, translateWithFormatting } from '@/utils/test/i18n';
+
+it('should render', () => {
```

Comment:
Can you wrap this in a describe so there's a bit more context when these fail

Reply:
Fair point, will do.